src/common/browser-info/get-browser-info.ts

```diff
@@ -0,0 +1,27 @@
+import BrowserInfo, { DeviceOS } from './browser-info';
+
+export default function getBrowserInfo(): BrowserInfo {
+    return {
+        color_depth: screen.colorDepth,
+        device_os: _getDeviceOs(),
+        java_enabled: navigator.javaEnabled(),
+        language: navigator.language || navigator.languages[0],
+        screen_height: screen.height,
+        screen_width: screen.width,
+        time_zone: new Date().toLocaleTimeString('en-us', {timeZoneName: 'short'}).split(' ')[2],
+        time_zone_offset: new Date().getTimezoneOffset().toString(),
+    };
+}
+
+function _getDeviceOs(): DeviceOS {
+    let deviceOS: DeviceOS = DeviceOS.Other;
+    if (navigator.userAgent.indexOf('Win') !== -1) {
```

Comment:
Since all we are doing here is parsing the `UserAgent`, should we just move all this logic to bigpay and figure out the device_os from there?

We can read the User Agent directly from the header in bigpay and work out what the value for device_os should be.